designs/lily/src/back.mjs

```diff
@@ -0,0 +1,577 @@
+      paths.hemBase.hide()
+    }
+
+    if (paperless) {
+      // Help construct cross seam
+      paths.hint = new Path()
+        .move(points.crossSeamCurveStart)
+        .line(points.crossSeamCurveMax)
+        .line(points.fork)
+        .attr('class', 'note lashed')
+      macro('hd', {
+        from: points.bottomIn,
+        to: points.bottomOut,
+        y: points.bottomIn.y - 30,
+      })
+      macro('hd', {
+        from: points.bottomIn,
+        to: points.bottom,
+        y: points.bottomIn.y - 15,
+      })
+      macro('hd', {
+        from: points.bottom,
+        to: points.bottomOut,
+        y: points.bottomIn.y - 15,
+      })
+      macro('vd', {
+        from: points.bottomOut,
+        to: points.styleWaistOutLily,
+        x:
+          (points.seatOut.x > points.styleWaistOutLily.x
+            ? points.seatOut.x
+            : points.styleWaistOutLily.x) +
+          sa +
+          15,
+      })
+      macro('vd', {
+        from: points.bottomIn,
+        to: points.fork,
+        x: points.fork.x - sa - 15,
+      })
+      macro('vd', {
+        from: points.fork,
+        to: points.styleWaistInLily,
+        x: points.fork.x - sa - 15,
+      })
+      macro('vd', {
+        from: points.bottomIn,
+        to: points.styleWaistInLily,
+        x: points.fork.x - sa - 30,
+      })
+      macro('vd', {
+        from: points.crossSeamCurveStart,
+        to: points.styleWaistInLily,
+        x: points.crossSeamCurveStart.x - sa - 15,
+      })
+      macro('hd', {
+        from: points.styleWaistInLily,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 15,
+      })
+      macro('hd', {
+        from: points.crossSeamCurveStart,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 30,
+      })
+      macro('hd', {
+        from: points.crossSeamCurveMax,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 45,
+      })
+      macro('hd', {
+        from: points.fork,
+        to: points.grainlineTop,
+        y: points.styleWaistInLily.y - sa - 60,
+      })
+      macro('hd', {
+        from: points.grainlineTop,
+        to: points.styleWaistOutLily,
+        y: points.styleWaistInLily.y - sa - 15,
+      })
+      if (points.seatOut.x > points.styleWaistOutLily.x) {
+        macro('hd', {
+          from: points.grainlineTop,
+          to: points.seatOut,
+          y: points.styleWaistInLily.y - sa - 30,
+        })
+      }
+    }
+  }
+
+  return part
+}
+
+export const back = {
+  name: 'lily.back',
+  from: titanBack,
+  //after: titanFront,
+  measurements: [
+    'ankle',
+    'heel', // secondary measurement, used instead of ankle
+  ],
+  options: {
+    fitGuides: { bool: false, menu: 'advanced' },
+    fitKnee: { bool: true, menu: undefined },
+    legBalance: 0.5, // between back and front parts
+    waistBalance: 0.5,
+    crotchDrop: { pct: 0, min: 0, max: 15, menu: 'advanced' }, // 'downgrade' to advanced menu
```

Comment:
Consider adding a `toAbs()` method here

Reply:
Since it's in the 'advanced' menu, I don't think it's worth it.